src/controllers/UserController.js

```diff
@@ -29,3 +37,45 @@ export const addUsers = async (req, res) => {
     res.status(500).json(err);
   }
 };
+
+/**
+ *
+ * @param {object} req - request object
+ *
+ * @param {object} res - response object
+ *
+ * @returns {obj} - it return a success message if the request is a login user
+ */
+export const editUserProfile = async (req, res) => {
+  const { firstName } = req.body;
+  const { id } = req.user.userData;
+  try {
+    const data = await userModel.update(`"firstName" = '${firstName}' WHERE "id" = ${id}`);
+    if (!data.rowCount) {
+      return res.status(400).json({ Message: 'Bad request' });
+    }
+    return res.status(200).json({ message: 'Profile updated successfully' });
+  } catch (error) {
+    res.json({ message: error.message });
+  }
+};
+
+/**
+ *
+ * @param {object} req - request
+ *
+ * @param {object} res - response
+ *
+ * @returns {obj} - all user data in the database
+ */
+export const getAllUser = async (req, res) => {
+  try {
+    const data = await userModel.select('*');
+    if (!data.rowCount) {
+      return res.status(400).json({ message: 'Bad request' });
```

Comment:
Stop returning the message for the status code, have an actual message that talked about what actually happened not not something like `Bad Request`